lib/mix/tasks/openapi.spec.json.ex

```diff
@@ -10,7 +10,7 @@ defmodule Mix.Tasks.Openapi.Spec.Json do
   defmodule Options do
     @moduledoc false
 
-    defstruct filename: nil, spec: nil, pretty: false
+    defstruct filename: nil, spec: nil, pretty: false, vendor_extensions: true
```

Comment:
Could you add some documentation for the options in the `@moduledoc` ?

Something like the style of the `ecto.migrate` task: https://github.com/elixir-ecto/ecto_sql/blob/master/lib/mix/tasks/ecto.migrate.ex#L72-L106

Reply:
@mbuhot Good idea. Pushed a commit.